docs/source/error_code_list2.rst

```diff
@@ -255,3 +255,31 @@ no error in practice. In such case, it might be prudent to annotate ``items: Seq
 This is similar in concept to ensuring that an expression's type implements an expected interface (e.g. ``Sized``),
 except that attempting to invoke an undefined method (e.g. ``__len__``) results in an error,
 while attempting to evaluate an object in boolean context without a concrete implementation results in a truthy value.
+
+
+Check that ``# type: ignore`` include an error code
+---------------------------------------------------
+
+Warn when an ``# type: ignore`` comment does not specify any error codes.
+This clarifies the intent of the ignore as well as ensuring that only the
+expected errors are silenced.
+
+Example:
+
+.. code-block:: python
+
+    # mypy: enable-error-code ignore-without-code
+
+    class Foo:
+        def __init__(self, name: str) -> None:
+            self.name = name
+
+    f = Foo('foo')
+
+    # This line has a typo that mypy can't help with as both the expected 'assignment' and 'attr-defined' are silenced
```

Comment:
```suggestion
    # This line has a typo that mypy can't help with as both the expected error 'assignment' and the unexpected error 'attr-defined' are silenced
```

This was a bit hard to read. Probably should also split it into multiple lines.

Reply:
7798cbd902b3c6c9a781eba1e5ac857ef5dbe355 hopefully helps here?